libraries/localization-utilities/src/schemas/locJson.schema.json

```diff
@@ -9,7 +9,7 @@
     }
   },
   "patternProperties": {
-    "^[A-Za-z_][0-9A-Za-z_]*$": {
+    "^[A-Za-z_$][0-9A-Za-z_$]*(?<!\\$schema)$": {
```

Comment:
This appears to mean that the string cannot match `\$schema` if I'm understanding it right? I think the `$` here may get parsed as the `EOL` matching pattern though?

Reply:
The `$` is escaped. I ended up changing this anyway.